sdk/docker/dockerSkeleton/server/src/service.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs');
+var spawn = require('child_process').spawn;
+
+function HelloService(rawLog, logger) {
+
+    var clientProg = "/blackbox/client/clientApp";
+    var server = undefined;
+
+    /**
+     * Starts the server.
+     *
+     * @param app express app
+     */
+    this.start = function start(app) {
+        var self = this;
+        server = app.listen(app.get('port'), function() {
+            var host = server.address().address;
+            var port = server.address().port;
+            logger.info('[start] listening at http://%s:%s', host, port);
+        });
+    }
+
+    /**
+     * req.body = { main: String, code: String, name: String }
+     */
+    this.initCode = function initCode(req, res) {
+        if (status === Status.ready) {
```

Comment:
looking at the test below, it's using one from dockerhub that's prebuilt. 
i think it would be better to do avoid that altogether perhaps, as you suggest.